src/main/java/com/dabsquared/gitlabjenkins/connection/GitLabConnection.java

```diff
@@ -277,28 +234,23 @@ public FormValidation doCheckReadTimeout(@QueryParameter Integer value) {
         public FormValidation doTestConnection(
                 @QueryParameter String url,
                 @QueryParameter String apiTokenId,
-                @QueryParameter String clientBuilderId,
                 @QueryParameter boolean ignoreCertificateErrors,
                 @QueryParameter int connectionTimeout,
                 @QueryParameter int readTimeout) {
             Jenkins.get().checkPermission(Jenkins.ADMINISTER);
             try {
-                new GitLabConnection(
-                                "",
-                                url,
-                                apiTokenId,
-                                clientBuilderId,
-                                ignoreCertificateErrors,
-                                connectionTimeout,
-                                readTimeout)
-                        .getClient(null, null)
+                new GitLabConnection("", url, apiTokenId, ignoreCertificateErrors, connectionTimeout, readTimeout)
+                        .getGitLabApi(null, null)
+                        .getUserApi()
                         .getCurrentUser();
                 return FormValidation.ok(Messages.connection_success());
             } catch (WebApplicationException e) {
                 return FormValidation.error(Messages.connection_error(e.getMessage()));
             } catch (ProcessingException e) {
                 return FormValidation.error(
                         Messages.connection_error(e.getCause().getMessage()));
```

Comment:
I don't think we need to catch `ProcessingException` anymore as this could only be thrown from RESTEasy I think.